### ISSUE TITLE
Crosswire PPA is Gone

### ISSUE THREAD

**Issue by compoundfraxure** [crosswire/xiphos#932]
It seems that the Ubuntu PPA for Xiphos has been removed since running apt update yields:
`Err:39 http://ppa.launchpad.net/unit193/crosswire/ubuntu bionic Release 404  Not Found [IP: 91.189.95.83 80]`
`Reading package lists... Done`                   
`E: The repository 'http://ppa.launchpad.net/unit193/crosswire/ubuntu bionic Release' no longer has a Release file.`

And going to https://launchpad.net/~unit193/+archive/ubuntu/crosswire directly also gives a 404 error.

**Comment by LAfricain** [comment]
Similar issue in #928

**Comment by unit193** [comment]
Howdy,

Yes, I removed the PPA as it was no longer being updated.  I've resigned from the pkg-crosswire team in Debian and decided not to maintain the PPA as I'd have to maintain different copies of all the source packages.

~Unit 193

**Comment by LAfricain** [comment]
I did it please test it https://launchpad.net/~lafricain79/+archive/ubuntu/lafricainenvrac

**Comment by davenorthcreek** [comment]
I got it to build on KUbuntu 18.04, I tried building from source but I'm not a guru and I didn't know how to encourage xiphos to use the sword lib I'd just built separately...
so I came back to @LAfricain's package and installed libbiblesync1.1 and libsword-1.8.1 and then xiphos came back to life.
I recommend pointing to @LAfricain 's package in the INSTALL doc so that more Ubuntu users use it and can help find issues.  Thanks @LAfricain  for putting this together!
